pgjdbc/src/test/java/org/postgresql/test/jdbc4/XmlTest.java

```diff
@@ -62,32 +62,47 @@ public XmlTest() throws Exception {
     _identityTransformer = factory.newTransformer();
   }
 
-  @Before
+  @Override
   public void setUp() throws Exception {
-    _conn = TestUtil.openDB();
-    Statement stmt = _conn.createStatement();
+    super.setUp();
+    assumeTrue(TestUtil.haveMinimumServerVersion(con, ServerVersion.v8_3));
+    assumeTrue("Server have been complied --with-libxml",isXmlEnabled(con));
```

Comment:
nit picking here, but could you change
assumeTrue("Server have been complied --with-libxml",isXmlEnabled(con));
to 
assumeTrue("Server has been complied --with-libxml",isXmlEnabled(con));

have -> has

Thanks